prow/config.yaml

```diff
@@ -26,13 +26,13 @@ presets:
         readOnly: true
   - labels:
       preset-sa-gke-kyma-integration: "true"
-    env: #TODO: unify/re-use values from other presets.
-      - name: GCLOUD_SERVICE_KEY_PATH
+    env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/credentials/sa-gke-kyma-integration/service-account.json
-      - name: GCLOUD_PROJECT_NAME
-        value: kyma-project
-      - name: GCLOUD_COMPUTE_ZONE
-        value: europe-west3-a
+      - name: CLOUDSDK_DNS_ZONE_NAME
+        value: "build-kyma" #GCloud DNS Zone Name (NOT it's DNS name!)
+      - name: SOURCES_DIR
```

Comment:
Note to reviewer:  I think this "SOURCES_DIR" is a value common to many jobs. I would like to extract it to a separate preset (to be reusable) but it's not purpose of _this_ PR, so for now I would just leave it here.

Reply:
I would rather call it KYMA_PROJECT_DIR

Reply:
Renamed